packages/custom_lint/lib/src/analyzer_plugin/lints.dart

```diff
@@ -155,9 +163,11 @@ final allLintsProvider =
         final previousLints = previous?.asData?.value;
         // We voluntarily treat "loading" as null, to clear lints during
         // plugin restart
-        final lints = next.isLoading ? null : next.asData?.value;
+        final lints =
+            next.isRefreshing || next.isLoading ? null : next.asData?.value;
         final allFiles = {...?lints?.keys, ...?previousLints?.keys};
 
+        print('Lints for file ${previous.runtimeType}');
```

Comment:
The previous / current is never loading or refreshing, even though I explicitly set the loading state on refresh above.

Reply:
Nevermind, it was, but the